src/3rdParty/_PCL/services/PointCloudUtils.jl

```diff
@@ -18,7 +18,7 @@ function getDataPointCloud(
       return nothing
     end
     # FIXME, change serialization to more standard pcd or laz formats, see Caesar.jl #921
-    return dbl |> IOBuffer |> Serialization.deserialize
```

Comment:
i'll tweak this elsewhere.  Issue is NCE and DFG are inconsistent on `getData`.  I did a bad job of that in the past and going go fix that proper quick, before it proliferates too much